tests/test_butlerUtils.py

```diff
@@ -376,7 +376,8 @@ def test_dafButlerRaiseTypes(self):
         # If DAF_BUTLER_REPOSITORY_INDEX is not set *at all* then
         # using an instrument label raises a FileNotFoundError
         with unittest.mock.patch.dict('os.environ'):
-            del os.environ['DAF_BUTLER_REPOSITORY_INDEX']
+            if 'DAF_BUTLER_REPOSITORY_INDEX' in os.environ:  # can't del unless it's already there
```

Comment:
It's more compact to say `os.environ.pop('DAF_BUTLER_REPOSITORY_INDEX', None)`, but it might be less clear.

Reply:
Yeah, I think I like the explicit way in this instance.